data/scrape/lib/languages.json

```diff
@@ -1,55 +1,33 @@
 {
-    "aar": {
-        "iso639_name": "Afar",
-        "wiktionary_name": "Afar",
-        "wiktionary_code": "aa",
-        "casefold": true,
-        "script": {
-            "latn": "Latin"
-        }
-    },
```

Comment:
I'm confused -- maybe I'm missing something. Why are many entries such as this one for `aar` being deleted in this JSON file?